dotcom-rendering/src/web/components/GetMatchTabs.tsx

```diff
@@ -0,0 +1,38 @@
+import { useApi } from '@root/src/web/lib/useApi';
+
+import { Placeholder } from '@root/src/web/components/Placeholder';
+
+import { MatchTabs } from './MatchTabs';
+
+type Props = {
+	matchUrl: string;
+	format: ArticleFormat;
+};
+
+const Loading = () => <Placeholder height={230} />;
```

Comment:
I suspect this value can be smaller. We know the height of `MatchTabs` so we can put that in here